src/main/resources/assets/appliedenergistics2/lang/de_de.lang

```diff
@@ -84,6 +85,7 @@ chat.appliedenergistics2.StationCanNotBeLocated=Station kann nicht lokalisiert w
 chat.appliedenergistics2.MachineNotPowered=Maschine bekommt keine Energie.
 chat.appliedenergistics2.CommunicationError=Fehler beim Kommunizieren mit dem Netzwerk.
 chat.appliedenergistics2.SavedSettings=Geräteeinstellungen auf Speicherkarte kopiert.
+chat.appliedenergistics2.ResetSettings=Geräteeinstellungen zurückgesetzt und auf Speicherkarte kopiert.
```

Comment:
That looks wrong, the en_us.lang says here "New device configuration created and copied to memory card." in German this would be "Neue Geräteeinstellungen erstellt und auf die Speicherkarte kopiert.".